app/views/catalog/_search_form.html.erb

```diff
@@ -0,0 +1,39 @@
+<%
+# Overridden from Blacklight to inject collection context into the search on the record view
+# and add a drop down to allow the user to choose to search w/i the collection or all collections
+%>
+<%= form_tag search_action_url, method: :get, class: 'search-query-form', role: 'search' do %>
+  <% if results_view? %>
+    <%= render_hash_as_hidden_fields(search_state.params_for_search.except(:q, :search_field, :qt, :page, :utf8, :within_collection)) %>
+  <% elsif record_view? %>
+    <%= render_hash_as_hidden_fields({ f: { collection_sim: [@document.collection_name] } }) %>
+  <% end %>
+  <div class="d-md-flex">
+    <%= render 'within_collection_dropdown' if record_view? || (results_view? && has_search_parameters?) %>
```

Comment:
I'm assuming that we only render the drop down in search results and the record view.  The search form also shows up on the home page and in the repositories view.  Should it be in all those places too?

Reply:
It is meant to be present on all pages, even though it can only have one choice (all collections) at the home and repositories pages. It is intended to also support future options that may be available from those locations, like "digital content".